x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/rules/read_rules_route.ts

```diff
@@ -41,8 +44,24 @@ export const createReadRulesRoute: Hapi.ServerRoute = {
         id,
         ruleId,
       });
+      const ruleStatuses = await savedObjectsClient.find({
+        type: ruleStatusSavedObjectType,
+        perPage: 10,
+        search: `"${id}"`,
```

Comment:
You want rule.id here instead of id as it is what was retrieved by either id or rue_id.